src/index.js

```diff
@@ -26,9 +46,36 @@ module.exports = (whitelist = []) => {
 
     traverse(clone).forEach(function() {
       if (!this.isLeaf) {
+
         return;
       }
 
+      if (typeof this.node === 'string' && this.node.includes('?')){
```

Comment:
That package is not usable since the url we receive is something like `/v0/users/?query=foobar` and [URL.js](https://www.npmjs.com/package/url-js) does not recognise it as an url